src/sentry/utils/email.py

```diff
@@ -446,17 +446,23 @@ def get_connection(fail_silently=False):
     )
 
 
-def send_mail(subject, message, from_email, recipient_list, fail_silently=False, **kwargs):
+def send_mail(
+    subject, message, from_email, recipient_list, fail_silently=False, headers=None, **kwargs
+):
     """
     Wrapper that forces sending mail through our connection.
     Uses EmailMessage class which has more options than the simple send_mail
     """
+    if headers is None:
+        headers = {}
```

Comment:
I'm not sure an empty dictionary will solve the issue you linked. That issue was also aiming to have auto-reply headers added, which doesn't seem to be part of this change.

Reply:
Forgot that bit while making the commit, my bad.